packages/block-editor/src/components/link-control/style.scss

```diff
@@ -268,7 +268,7 @@ $block-editor-link-control-number-of-actions: 1;
 			font-weight: normal;
 		}
 
-		svg {
+		.components-external-link__icon {
```

Comment:
This element also has "Opens in a new tab" as an `aria-label` for screen readers to announce that it's an external link, and if it's set to `display: none`, it doesn't get announced. Might need to raise the specificity with `.components-external-link__icon svg`?

Reply:
There's not an SVG anymore; that's the issue.

![CleanShot 2024-04-10 at 17 45 44](https://github.com/WordPress/gutenberg/assets/1813435/7ff4e9f1-c008-4b76-a7a9-a254377965f7)

This is what we had before https://github.com/WordPress/gutenberg/pull/60255:

![CleanShot 2024-04-10 at 17 47 09](https://github.com/WordPress/gutenberg/assets/1813435/cff3dbfc-fd96-4cb1-b939-9fdd583b20e8)

Reply:
@jameskoster any ideas here?